### PR TITLE
core: add fsid to the additionalPrinterColumns on cephcluster custom resource

### PR DESCRIPTION
**Pull request by thenamehasbeentake**
**Description of your changes:**
Add fsid to the  additionalPrinterColumns on the crd types

**Which issue is resolved by this Pull Request:**
[11862](https://github.com/rook/rook/issues/11862)

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: pkg/apis/ceph.rook.io/v1/types.go [hunk 1]
```diff
@@ -42,6 +42,7 @@ import (
 // +kubebuilder:printcolumn:name="Message",type=string,JSONPath=`.status.message`,description="Message"
 // +kubebuilder:printcolumn:name="Health",type=string,JSONPath=`.status.ceph.health`,description="Ceph Health"
 // +kubebuilder:printcolumn:name="External",type=boolean,JSONPath=`.spec.external.enable`
+// +kubebuilder:printcolumn:name="FSID",type=string,JSONPath=`.status.ceph.fsid`,description="Ceph Fsid"
```
Comment (parth-gr) [inline pkg/apis/ceph.rook.io/v1/types.go:45]: +kubebuilder:printcolumn:name="FSID",type=string,JSONPath=`.status.ceph.fsid`,description="Ceph FSID"

File: pkg/apis/ceph.rook.io/v1/types.go [hunk 2]
```diff
@@ -42,6 +42,7 @@ import (
 // +kubebuilder:printcolumn:name="Message",type=string,JSONPath=`.status.message`,description="Message"
 // +kubebuilder:printcolumn:name="Health",type=string,JSONPath=`.status.ceph.health`,description="Ceph Health"
 // +kubebuilder:printcolumn:name="External",type=boolean,JSONPath=`.spec.external.enable`
+// +kubebuilder:printcolumn:name="FSID",type=string,JSONPath=`.status.ceph.fsid`,description="Ceph FSID"
```
Comment (travisn) [inline pkg/apis/ceph.rook.io/v1/types.go:45]: I don't believe you can have a space in the name.
```suggestion
// +kubebuilder:printcolumn:name="FSID",type=string,JSONPath=`.status.ceph.fsid`,description="FSID"
```

This is what I see what I tested your change:
```
% kn get cephcluster
NAME         DATADIRHOSTPATH   MONCOUNT   AGE   PHASE   MESSAGE                        HEALTH      EXTERNAL   FSID
my-cluster   /var/lib/rook     3          42h   Ready   Cluster created successfully   HEALTH_OK              9341d8c4-06e8-4c24-9451-dffd04600cac
```

Comment (thenamehasbeentake) [reply pkg/apis/ceph.rook.io/v1/types.go:45]: `"Ceph FSID"` is description, not name

Comment (travisn) [reply pkg/apis/ceph.rook.io/v1/types.go:45]: Ha! I overlooked that name. LGTM

**Review by travisn — APPROVED** [review]

**Pull request merged**